src/components/transactions/attachment-row.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTxState } from '@common/hooks/use-tx-state';
+import { Flex, Box, Text } from '@stacks/ui';
+import React from 'react';
+
+function readable(data: string)
+	{
+	const text = Buffer.from(data,'hex').toString('ascii');
```

Comment:
is it an ascii string or utf8?

Reply:
I try to force it as ASCII to see it is readable, but we can also try utf8.